src/components/DirectLinks/DirectLinks.constants.js

```diff
@@ -0,0 +1,18 @@
+export const LINKS = {
+  MIJN_BELASTINGEN: {
+    displayName: 'Mijn belastingen',
+    url: 'https://belastingbalie.amsterdam.nl',
+  },
+  // MIJN_PARKEREN: {
+  //   displayName: 'Mijn parkeren',
+  //   url: '',
+  // },
+  // MIJN_WERK_EN_INKOMEN: {
```

Comment:
this link? `https://www.amsterdam.nl/werk-inkomen` I guess placeholders are fine at this stage.

Reply:
Got the correct url, for now, from Ali. I will add it.